### PR TITLE
Allow customization of the prefix character (~)

### PR DESCRIPTION
**Pull request by jesstelford**
Enables users to specify paths prefixed by whatever character they want:

```node
var sass = require('node-sass');
var packageImporter = require('node-sass-package-importer');

var options = {
  prefix: '+'
};

sass.render({
  ...
  importer: packageImporter(options)
  ...
});
```
```scss
@import '+bootstrap';
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -43,6 +43,7 @@ In the example below you can see the default configuration options.
 - `cwd`: Defines the path in which your `node_modules` directory is found.
 - `extensions`: Set which file extensions should get resolved.
 - `packageKeys`: You can define the `package.json` keys to search for and in which order.
+- `leader`: You can set the special character for indicating a module resolution.
```
Comment (maoberlehner) [inline README.md]: Would you mind to change this to `prefix`?

File: js/lib/PackageImporter.js [hunk 2]
```diff
@@ -25,17 +25,24 @@ export default class PackageImporter {
...
      */
     this.options = Object.assign({}, defaultOptions, options);
+
+    /**
+     * Ensure any regex characters entered are escaped
+     */
+    this.options.leader = this.options.leader.replace(/[-/\\^$*+?.()|[\]{}]/g, `\\$&`);
+
     /**
      * Match tilde symbol at the beginning of urls (except posix home "~/" directory).
```
Comment (maoberlehner) [inline js/lib/PackageImporter.js:42]: Please update this description too.

**Review by maoberlehner — CHANGES_REQUESTED** [review]
Thank you for your contribution. I added some little suggestions for small fixes.

Please run `npm run scripts` and also commit the generated dist files.

Thank you.

File: README.md (hunk 1 above)
Comment (jesstelford) [reply README.md]: Done 👍

File: js/lib/PackageImporter.js (hunk 2 above)
Comment (jesstelford) [reply js/lib/PackageImporter.js:42]: Done 👍

**Comment by jesstelford** [comment]
`leader` has been changed to `prefix`. Thanks for the excellent library! :)

*edit*: And I've also committed the `dist` files 👍

**Comment by jesstelford** [comment]
Thanks for the really quick merge :)

**Comment by maoberlehner** [comment]
No problem. I also created a new tag and published 3.0.1 on npm.

Thank you for your contribution.

**Pull request merged**